code/game/objects/items/weapons/tools/crowbars.dm

```diff
@@ -7,7 +7,6 @@
 	force = WEAPON_FORCE_PAINFUL
 	worksound = WORKSOUND_EASY_CROWBAR
 	w_class = ITEM_SIZE_NORMAL
-	reduced_storage_cost = TRUE //It's long and thin so it doesn't grow exponentially
```

Comment:
This is what you plan with this crowbar taking more space than now ?

Reply:
Change was intended, but I can reduce the crowbars size.

Reply:
ITEM_SIZE_SMALL means you can put your crowbar into your ear slot? So this would break crowbars a little.